lib/shoryuken/environment_loader.rb

```diff
@@ -99,7 +99,7 @@ def prefix_active_job_queue_names
     end
 
     def parse_queue(queue, weight = nil)
-      [weight.to_i, 1].max.times { Shoryuken.queues << queue }
+      Shoryuken.add_queue(queue, [weight.to_i, 1])
```

Comment:
@h3poteto nice! thanks for reviewing it. I meant `[weight.to_i, 1].max`.

This code needs a test `rspec`. Not sure how tests are passing, even the integration.